specification.md

```diff
@@ -1488,8 +1490,12 @@ a starting [Event state](#Event-state), it is not used to validate its event pay
 
 The `start` property defines the workflow starting information. For more information see the [start definition](#Start-Definition) section.
 
-The `schemaVersion` property can be used to set the specific Serverless Workflow schema version to use
-to validate this workflow markup. If not provided the latest released schema version is assumed.
+The `specVersion` property can be used to set the specific Serverless Workflow specification release version 
+the workflow markup adheres to. The default value of this property is `"latest"`, meaning that 
```

Comment:
well it means that if you have a workflow definition where you don't specify the specVersion and execute it on runtimes A and B, both would parse it against the latest specification release schemas. On our website we provide already:

https://serverlessworkflow.io/schemas/latest/workflow.json

so that can map the "latest" default keyword